rpm.mk

```diff
@@ -124,8 +142,11 @@ rpmbuildprep:
 	if [ $(BUNDLE_JEMALLOC) -eq 1 ]; then \
 		cp dist/sources/$(JEMALLOC_TARBALL) $(RPMBUILD)/SOURCES/ ; \
 	fi
+	if [ $(BUNDLE_LIBDB) -eq 1 ]; then \
+		cp dist/sources/$(LIBDB_NAME).tar.bz2 $(RPMBUILD)/SOURCES/ ; \
+	fi
 
-srpms: rpmroot srpmdistdir download-cargo-dependencies tarballs rpmbuildprep 
+srpms: rpmroot srpmdistdir download-cargo-dependencies tarballs rpmbuildprep
```

Comment:
The resulting srpm file doesn't contain libdb tarballs or patches and fails to rebuild.

Reply:
Adding the url in spec file Sources solved the issue.